src/ShellCheck/Interface.hs

```diff
@@ -221,7 +221,7 @@ newCheckDescription = CheckDescription {
     }
 
 -- Supporting data types
-data Shell = Ksh | Sh | Bash | Dash deriving (Show, Eq)
+data Shell = Ksh | Sh | Bash | Dash | Busyboxsh deriving (Show, Eq)
```

Comment:
`BusyboxSh` would be easier to read.

Any thoughts on `BusyboxSh` vs `BusyboxAsh`? I have no strong opinions either way.

Reply:
I went for busybox sh, as it seems to strongly diverge from other Ash implementations.

Reply:
renamed it to `BusyboxSh`